packages/universal-test-adapter-jest/src/index.ts

```diff
@@ -48,6 +50,20 @@ export async function executeTests({ testsToRun = [] }: AdapterInput): Promise<A
     args.push('--testNamePattern', `${describeIts.join('|')}`)
   }
 
+  switch (reportFormat) {
+    case 'default':
```

Comment:
minor: include junit as a case here so it's less confusing from a user perspective that only wants to use junit.
We can hold off on this until we actually add more formats though, doesn't matter too much atm

Reply:
Hm yeah good thought, do you have any idea how easy it is to generate junit reports for dotnet? That's the main reason I left it out since I assumed default report format for dotnet would not be junit -- but if it's easy we should add it. Will have a closer look and change it up in the follow-up for adding pytest junit reports.